spec/Section 6 -- Execution.md

```diff
@@ -143,6 +147,96 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
     selection set.
   * Return an unordered map containing {data} and {errors}.
 
+### Subscription
+
+Unlike queries and mutations, subscriptions have a lifetime that consists of three
```

Comment:
If this sentence is going to start with "unlike", it should probably provide the thing we are comparing to, like:

> Unlike queries and mutations, which deal with one request and response, subscriptions have a lifetime that consists of three